### PR TITLE
Add hooks to the module template.

### PR DESCRIPTION
**Pull request by Yajo**
Another :gift: for the template.

### CODE REVIEW

File: template/module/hooks.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# © <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (pedrobaeza) [inline template/module/hooks.py]: Change this to the new shorter form :wink:

Comment (Yajo) [inline template/module/hooks.py]: Yikes, true!

**Comment by pedrobaeza** [comment]
:+1: 

(coveralls has turned back to the ugly option of putting a comment with the coverage. I hate this...)

**Comment by lmignon** [comment]
:+1:

**Comment by osvalr** [comment]
late but :+1:

**Pull request merged**